test/TimeoutTest.php

```diff
@@ -26,6 +26,7 @@ public function setUp(): void
         parent::setUp();
 
         $this->client = new Client;
+        $this->client->setRetryLimit(0);
```

Comment:
I think we shouldn't retry on timeouts, so this test should be unaffected.

Reply:
Really? Request timeouts are the thing I see most often in my logs for request failures.

Reply:
I think we should use the configured timeouts and not multiply these by X by default. The reason for including a retry by default is mostly due to connections closing unexpectedly.